README.md

```diff
@@ -1,17 +1,22 @@
-[![NPM][npm]][npm-url]
-[![Deps][deps]][deps-url]
-[![Tests][travis]][travis-url]
-[![Coverage][cover]][cover-url]
+[![npm][npm]][npm-url]
+[![node][node]][node-url]
```

Comment:
I in favour of leaving them, because the 'standard' badges are mostly consistent the posthtml org repos.